apps/namehashlabs.org/components/2 - molecules/product-component.tsx

```diff
@@ -62,12 +62,15 @@ export const ProductComponent = ({
             <SectionText className="lg:text-start text-center w-full text-lg leading-7 font-light">
               <Balancer>{subtitle}</Balancer>
             </SectionText>
+
             {buttonUrl && (
-              <Link href={buttonUrl} asChild>
-                <IconButton>
-                  Join the discussion <Link.ExternalIcon />
+              <div className="flex justify-center lg:justify-start">
+                <IconButton asChild>
```

Comment:
Something funny is happening here. Perhaps it's because this is using `IconButton` and not `Button`?

Here's how this looks now:

![CleanShot 2024-09-21 at 00 08 41](https://github.com/user-attachments/assets/88da1bb3-e586-47e2-8a41-d25bcea469c3)

And here's how it looked before:

![CleanShot 2024-09-21 at 00 09 06](https://github.com/user-attachments/assets/dd7c5593-7868-4642-895a-e2da4d7d6549)

Reply:
@lightwalker-eth It seems we have different styles of the external link icon. We should settle for one, wdyt?